src/scandeval/benchmarker.py

```diff
@@ -548,8 +548,8 @@ def model_has_been_benchmarked(
         if (
             record.model == model_id
             and record.dataset == dataset
-            and record.few_shot == few_shot
             and record.validation_split == validation_split
+            and (not record.generative or record.few_shot == few_shot)
```

Comment:
Quite a long conditional. Might be relevant to name element of it to make it more clear what each segment check. (it is not too bad as the function names the overall conditional)